src/briefcase/platforms/android/gradle.py

```diff
@@ -212,20 +213,21 @@ def run_app(self, app: BaseConfig, device_or_avd=None, **kwargs):
         with self.input.wait_bar("Installing new app version..."):
             adb.install_apk(self.binary_path(app))
 
-        self.logger.info("Starting app...", prefix=app.app_name)
-        with self.input.wait_bar("Clearing device log..."):
-            adb.clear_log()
```

Comment:
Now that we're filtering by PID, there's no need to clear the log to exclude earlier instances of the same app. And leaving the log intact allows the user to go back and examine it later with `adb` or Android Studio.

The PID filtering is done within the device and causes no noticable delay, even when the log buffer contains thousands of messages.